<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="41" started-at="2017-02-22T15:55:44Z" finished-at="2017-02-22T15:55:44Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="41" started-at="2017-02-22T15:55:44Z" finished-at="2017-02-22T15:55:44Z">
      <class name="com.vitalii.vitaliiFramework2.tests.LoginPageTestsWD">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:com.vitalii.vitaliiFramework2.tests.LoginPageTestsWD@42e26948]" name="beforeSuite" is-config="true" duration-ms="7" started-at="2017-02-22T17:55:44Z" finished-at="2017-02-22T17:55:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="FAIL" signature="doSuccessfulLoginTest()[pri:0, instance:com.vitalii.vitaliiFramework2.tests.LoginPageTestsWD@42e26948]" name="doSuccessfulLoginTest" duration-ms="22" started-at="2017-02-22T17:55:44Z" finished-at="2017-02-22T17:55:44Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[This test has been skipped]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: This test has been skipped
	at com.vitalii.vitaliiFramework2.listeners.TestResultListener.onTestSkipped(TestResultListener.java:36)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1653)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1645)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:703)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:970)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1149)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:75)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:46)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doSuccessfulLoginTest -->
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.vitalii.vitaliiFramework2.tests.LoginPageTestsWD@42e26948]" name="beforeTest" is-config="true" duration-ms="0" started-at="2017-02-22T17:55:44Z" finished-at="2017-02-22T17:55:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.vitalii.vitaliiFramework2.tests.LoginPageTestsWD@42e26948]" name="beforeTest" is-config="true" duration-ms="0" started-at="2017-02-22T17:55:44Z" finished-at="2017-02-22T17:55:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.vitalii.vitaliiFramework2.tests.LoginPageTestsWD@42e26948]" name="beforeTest" is-config="true" duration-ms="1" started-at="2017-02-22T17:55:44Z" finished-at="2017-02-22T17:55:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="doSuccessfulLoginTestAndPublishSignupForm()[pri:0, instance:com.vitalii.vitaliiFramework2.tests.LoginPageTestsWD@42e26948]" name="doSuccessfulLoginTestAndPublishSignupForm" duration-ms="2" started-at="2017-02-22T17:55:44Z" finished-at="2017-02-22T17:55:44Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.vitalii.vitaliiFramework2.helpers.Configurations.getEnvironmentUrl(Configurations.java:37)
	at com.vitalii.vitaliiFramework2.pages.LoginPage.openLoginPage(LoginPage.java:14)
	at com.vitalii.vitaliiFramework2.pages.LoginPage.doSuccessfulLogin(LoginPage.java:26)
	at com.vitalii.vitaliiFramework2.tests.LoginPageTestsWD.doSuccessfulLoginTestAndPublishSignupForm(LoginPageTestsWD.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:75)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:46)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doSuccessfulLoginTestAndPublishSignupForm -->
        <test-method status="FAIL" signature="clearCookies()[pri:0, instance:com.vitalii.vitaliiFramework2.tests.LoginPageTestsWD@42e26948]" name="clearCookies" is-config="true" duration-ms="0" started-at="2017-02-22T17:55:44Z" finished-at="2017-02-22T17:55:44Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.vitalii.vitaliiFramework2.driver.WebDriverThread.<init>(WebDriverThread.java:17)
	at com.vitalii.vitaliiFramework2.driver.DriverFactory$1.initialValue(DriverFactory.java:28)
	at com.vitalii.vitaliiFramework2.driver.DriverFactory$1.initialValue(DriverFactory.java:1)
	at java.lang.ThreadLocal.setInitialValue(ThreadLocal.java:180)
	at java.lang.ThreadLocal.get(ThreadLocal.java:170)
	at com.vitalii.vitaliiFramework2.driver.DriverFactory.getDriver(DriverFactory.java:50)
	at com.vitalii.vitaliiFramework2.driver.DriverFactory.clearCookies(DriverFactory.java:59)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:216)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:712)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:75)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:46)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clearCookies -->
      </class> <!-- com.vitalii.vitaliiFramework2.tests.LoginPageTestsWD -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
